app/models/agents/tumblr_likes_agent.rb

```diff
@@ -0,0 +1,58 @@
+module Agents
+  class TumblrLikesAgent < Agent
+    include TumblrConcern
+
+    gem_dependency_check { defined?(Tumblr::Client) }
+
+    description <<-MD
+      The Tumblr Likes Agent checks for liked Tumblr posts from a specific blog.
+
+      #{'## Include `tumblr_client` and `omniauth-tumblr` in your Gemfile to use this Agent!' if dependencies_missing?}
+
+      To be able to use this Agent you need to authenticate with Tumblr in the [Services](/services) section first.
+
+
+      **Required fields:**
+
+      `blog_name` The Tumblr URL you're querying (e.g. "staff.tumblr.com")
+
+      Set `expected_update_period_in_days` to the maximum amount of time that you'd expect to pass between Events being created by this Agent.
+    MD
+
+    default_schedule 'every_1h'
+
+    def validate_options
+      errors.add(:base, 'blog_name is required') unless options['blog_name'].present?
+      errors.add(:base, 'expected_update_period_in_days is required') unless options['expected_update_period_in_days'].present?
+    end
+
+    def working?
+      event_created_within?(interpolated['expected_update_period_in_days']) && !recent_error_logs?
+    end
+
+    def default_options
+      {
+        'expected_update_period_in_days' => '10',
+        'blog_name' => '{{blog_name}}',
```

Comment:
You probably don't need the Liquid interpolation here since your Agent can't receive Events. This could be just `'blog_name' => 'staff'` or something for the `default_options`.